### DISCUSSION TITLE
Query included relationships

### DISCUSSION THREAD

**Discussion by chrisidakwo** [spatie/laravel-query-builder#496] [Q&A] [answered]
How can one easily query a relationship just like Laravel provides? Something like the image below:

![carbon(6)](https://user-images.githubusercontent.com/29854560/87586853-2178f700-c6d9-11ea-8e9c-eaf36c01bd79.png)

**Comment by dominikb** [comment, marked as answer]
Have a look at this discussion: #499 
It treats a similar problem.

As far as I know, this behavior can not be achieved directly using this package but rather the built-in methods on the underlying `QueryBuilder` instance.

Something like this could work:
```php
QueryBuilder::for(Addon::class)
    ->allowedIncludes('subscriptions')
    ->whereHas('subscription', fn($builder) => $builder->where('addon_subscriptions.organization_id', $this->organizationId))
    ->get();
```

**Reply by chrisidakwo** [reply]
Oh my! Such a simple solution. I didn't even think of that. Thanks! I found a way around it though. But I think this is more straightforward and "Laravely". Thank you!
